data/common.yaml

```diff
@@ -1 +1,35 @@
---- {}
+---
+profile_gsi_ssh::server::cert::files: {}
+profile_gsi_ssh::server::cert::hostcert: ""  # PROVIDE CONTENTS OF HOST IGTF CERTIFICATE
+profile_gsi_ssh::server::cert::hostkey: ""   # PROVIDE CONTENTS OF HOST PRIVATE KEY
+profile_gsi_ssh::server::cert::monitor: true
+profile_gsi_ssh::server::cert::monitor_file_content: |
+  [[inputs.x509_cert]]
+    sources = [
+    ]
+    interval = "3h"
+profile_gsi_ssh::server::cert::monitor_file_path: "/etc/telegraf/telegraf.d/sslcert-check.conf"
+profile_gsi_ssh::server::cert::monitor_interval: "3h"
+
+profile_gsi_ssh::server::allow_groups: []
+
+profile_gsi_ssh::server::config:
+  AllowGroups: "root"
+  AuthenticationMethods: "none"
+  PermitRootLogin: "no"
+
+profile_gsi_ssh::server::config_file: "/etc/gsissh/sshd_config"
+
+profile_gsi_ssh::server::deny_groups:
+  - "all_disabled_usr"
+
+profile_gsi_ssh::server::disabled_service_name: ""
+
+profile_gsi_ssh::server::gridmap::crons: {}
+profile_gsi_ssh::server::gridmap::files: {}
```

Comment:
Would be nice if the cron scripts were a template in ncsa/puppet-profile_gsi_ssh. Or even just take what's being set in hiera over in delta-control and put that here? Mainly so that when one project notices an improvement to the scripts it can be pushed up to here so other projects can see and bring down the change.

Unless we expect these scripts to all be very different project to project, which may be the case here.

Reply:
I agree, but currently this is very ad hoc depending on the project. And honestly we probably have not yet settled on the final method for Delta yet.

How about we add that as an issue to address/include in the future?

Reply:
OK, sure. I'll at least include slightly altered versions of what we're starting with for Delta.

I may opt to include them only as commented out hiera data.